koku/reporting/migrations/0160_gcp_compute_summary.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.5 on 2021-01-14 21:13
```

Comment:
I want to make sure this is the right way to do the migration I think something weird happened last time so I wanted to leave a note to double check.